.github/workflows/tag-and-publish.yml

```diff
@@ -108,7 +109,7 @@ jobs:
           git checkout main
           ls -la
           HELM_CUR_TAG=$(grep "/cbas:" terra-batch-libchart/values.yaml | sed "s,.*/cbas:,,")
-          HELM_NEW_TAG=${{ steps.tag.outputs.TAG }}
+          HELM_NEW_TAG=${{ steps.tag.outputs.tag }}
```

Comment:
L120 should now use the JIRA_ID from `ensure-jira-id` step

Reply:
Oops! Thanks!